### PR TITLE
Admin business client reports

### PR DESCRIPTION
**Pull request by iAmNikola**
- Renames Review to Report
- Implements logic in backend to support reviewing business client reports
- Created page for admin to see business client reports table with pagination support

### CODE REVIEW

File: backend/src/main/java/team10/app/repository/ReportRepository.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.data.jpa.repository.Query;
+import team10.app.model.Client;
+import team10.app.model.DeletionRequest;
+import team10.app.model.Report;
+
+import java.util.UUID;
+
+public interface ReportRepository extends JpaRepository<Report, UUID> {
+
+    Page<Report> findAll(Pageable pageable);
```
Comment (Ikkjo) [inline backend/src/main/java/team10/app/repository/ReportRepository.java:15]: Exists as predefined function so this is not needed

File: backend/src/main/java/team10/app/service/ReportService.java [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        reportRepository.saveAndFlush(report);
+        return reportDto;
+    }
+
+    public Page<Report> getNotReviewedReports(String sort, int page, int size) {
+        Pageable paging = PageRequest.of(page, size, Sort.by(reportGetSort(sort)));
+        return reportRepository.findAllNotReviewed(paging);
+    }
+
+    private List<Sort.Order> reportGetSort(String sort) {
```
Comment (Ikkjo) [inline backend/src/main/java/team10/app/service/ReportService.java:58]: Nije baš jasno šta funkcija treba da radi, možda može biti malo više čitko. Jel treba da vrati po čemu treba da se sortira?

**Review by Ikkjo — CHANGES_REQUESTED** [review]
Should fix according to comments, all else lgtm.

File: backend/src/main/java/team10/app/repository/ReportRepository.java (hunk 1 above)
Comment (iAmNikola) [reply backend/src/main/java/team10/app/repository/ReportRepository.java:15]: Its also not in use. Deleting it

**Pull request merged**